sklearn/cluster/tests/test_k_means.py

```diff
@@ -621,19 +591,28 @@ def test_score_max_iter(Estimator):
 @pytest.mark.parametrize(
     "array_constr", [np.array, sp.csr_matrix], ids=["dense", "sparse"]
 )
-@pytest.mark.parametrize("dtype", [np.float32, np.float64])
-@pytest.mark.parametrize("init", ["random", "k-means++"])
```

Comment:
The `init` is irrelevant to this test. I removed the parametrisation and just used "random". "k-means++" was actually the main reason these tests were slow